src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.css

```diff
@@ -20,3 +20,26 @@
   margin-top: 0.5em;
   font-size: small;
 }
+
+/*Style for the search box in the tree-node accordion on the left side*/
+:host >>> .gb-tree-filter-input > input {
+  font-size: small !important;
+  height: 1.8em;
+  width: 100%;
+  min-width: 3em;
+  max-width: 7em;
+}
+
+/*tree node styles for filtering*/
+:host >>> .gb-highlight-treenode {
```

Comment:
looks like a duplication - this class is already defined in gb-side-panel.component.css

Reply:
move tree css to the tree search component folder